src/components/tabs/Tabs.tsx

```diff
@@ -22,11 +25,70 @@ export const Tabs: ReactTabsFunctionComponent<TabsProps> = ({ children, ...other
 Tabs.tabsRole = "Tabs";
 
 // TabList
-export const TabList: ReactTabsFunctionComponent<TabListProps> = ({ children, ...otherProps }) => (
-  <RTabList className={styles.tabList} {...otherProps}>
-    {children}
-  </RTabList>
-);
+export const TabList: ReactTabsFunctionComponent<TabListProps> = ({ children, ...otherProps }) => {

```

Comment:
The overflow wrapper has a lot of similarities of the `TableWrapper`. This one seems to work a lot better (on desktop, at least) than the `TableWrapper` due to it not having the `touchScreen` conditional rendering.

Also, seeing this, I'm thinking maybe the other component shouldn't be called a `TableWrapper` but a `HorizontalOverflowWrapper`. That way, we could've simply wrapped the `TabsList` in the `HorizontalOverflowWrapper` and not have to duplicate any logic.

The `Table` instances could also still make use of the `HorizontalOverflowWrapper`. What do you think?

In the future we could even rename it to `OverflowWrapper` when we also add vertical features.